constitution.txt

```diff
@@ -34,20 +34,20 @@ Often, these are not necessarily bad, they are just not in the scope of our comm
  
 ### Submission guidelines
  
-	* Submissions should be on topic and fit an appropriate tag(s). If a tag doesn't fit, it might not be appropriate to post to Lobsters.
+	* Submissions should be on topic and fit an appropriate tag(s). If no tags fit, it might not be appropriate to post to Lobsters.
 	* Use original titles sans extraneous bits on submissions--leave editorializing to suggestions. For content older than a few years, suffix the title with the year of publication in parens.
 	* Add context to submissions when the context is not clear from the title.
 	* Do not submit spam or low-value articles.
 	* Similar submission may be merged by mods. Message or comment if you think a submission could be merged.
-	* For discussion stories, consider asking open-ended questions over ones with concrete solutions.
+	* For discussion stories, prefer asking open-ended questions over ones with concrete solutions.
```

Comment:
If could even be changed to something like, "Ask open-ended questions over ones with concrete solutions to engender valuable discussions." if people wanted to avoid technical support questions with single solutions. It just depends on what people want lobsters to be.